src/lib/VariableResolver.ts

```diff
@@ -5,6 +5,8 @@ export class VariableResolver
 {
     protected expressionRegex: RegExp = /\$\{(.*?)\}/gm;
     protected workspaceRegex: RegExp = /workspaceFolder\[(\d+)\]/gm;
+    protected configVarRegex: RegExp = /config:(.+)/m;
+    protected envVarRegex: RegExp = /env:(.+)/m;
```

Comment:
This both regexes would match something set as `config:something` and I believe the intention would be to match only `${config:something}`

Maybe you could use the `expressionRegex` as a model to work this way

Reply:
Those regexes operate only on values already matched by expressionRegex group, so basically, they operate on substrings like "config:something", "env:something".